stores/identity.ts

```diff
@@ -142,14 +143,19 @@ export const useIdentityStore = defineStore('identity', {
       const { isTestnet } = usePrefix()
       const { multiBalanceAssets, multiBalanceAssetsTestnet }
         = storeToRefs(useIdentityStore())
-      const { availableChainsByVm } = useChain()
+      const { vm } = useChain()
+
+      // useChain().availableChainsByVm excludes disabled chains like dot
+      const vmChains = chainList().filter(
```

Comment:
Should be computed or?

Reply:
`vm` is a computed 

![CleanShot 2024-08-13 at 09 53 42@2x](https://github.com/user-attachments/assets/781a59e1-80f9-416e-8881-08a7f3306a40)